src/components/Lesson/Lesson.js

```diff
@@ -103,6 +103,8 @@ const Lesson = React.createClass({
     //this.setLanguage();
   },
   componentDidMount() {
+    document.title = this.getTitle() === ''
+      ? 'Oppgave' : this.getTitle() + ' | Kodeklubben';
```

Comment:
Forøvrig, dersom du skal ha en "default" tekst som "Oppgave", er det ikke bedre å la "getTitle" returnere det? Så kan du la det stå `document.title = this.getTitle() + ' | Kodeklubben'
Dessuten hadde det nok vært fint det andre stedet getTitle brukes.